modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -765,8 +764,8 @@ func (suite *InterchainAccountsTestSuite) TestABCICodeDeterminism() {
 		},
 	}
 
-	hash := tmstate.ABCIResponsesResultsHash(&responses)
-	differentHash := tmstate.ABCIResponsesResultsHash(&differentResponses)
+	hash := ibctesting.ABCIResponsesResultsHash(&responses)
+	differentHash := ibctesting.ABCIResponsesResultsHash(&differentResponses)
```

Comment:
hmm, the purpose of this test was to test the value returned by tendermint `ABCIResponseResultsHash`. If we mock this function, that defeats the purpose.

Reply:
`ABCIResponsesResultHash` got removed in 0.35, but we are just replicating [its functionality from 0.34](https://github.com/tendermint/tendermint/blob/v0.34.19/state/store.go#L356-L358) into our own function.

Reply:
What was it replaced by? We will need to ensure the tx response is still included in the consensus hash, otherwise we need to remove inclusion of the transaction response in the interchain account acknowledgement

Reply:
If the transaction response is not included in the consensus hash, the tests should break, but if we mock the behaviour that they are included, we will never know if tendermint changes this functionality

Reply:
@colin-axner [refactored the tests](https://github.com/cosmos/ibc-go/pull/1349).